yt_dlp/extractor/zeenews.py

```diff
@@ -0,0 +1,57 @@
+from .common import InfoExtractor
+from ..utils import unified_strdate
+
+
+class ZeeNewsIE(InfoExtractor):
+
+    _VALID_URL = r'''(?x)
+                    (?:
+                    zeenews:|
+                    https?://(?:zeenews\.)?india\.com/(?:[^#?]+/)(?:video)/)
+                    (?P<display_id>[^#/?]+)/(?P<id>[^#/?]+)/?(?:$|[?#])
+                    '''
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:zeenews\.)?india\.com/[^#?]+/video/(?P<display_id>[^#/?]+)/(?P<id>[^#/?]+)/?(?:$|[?#])'
```

1. Use `display_id`
1. Add test case without `zeenews.`
1. Add test case for non-number `id`

Reply:
r'https?://zeenews\.india\.com/[^#?]+/video/(?P<display_id>[^#/?]+)/(?P<id>[0-9]+)'

?